lib/alex_koin/slack.ex

```diff
@@ -51,7 +70,7 @@ defmodule AlexKoin.SlackRtm do
     {"Created a new coin: `#{coin.hash}` with origin: '#{coin.origin}'", nil}
   end
   defp create_reply(user, _message, {:transfer, text}) do
-    regex = ~r/ transfer (?<coin_uuid>[0-9a-zA-Z-]+) to <@(?<to_slack_id>[A-Z0-9]+)> (?<memo>.*)/
+    regex = ~r/transfer (?<coin_uuid>[0-9a-zA-Z-]+) to <@(?<to_slack_id>[A-Z0-9]+)> (?<memo>.*)/
     captures = Regex.named_captures(regex, text)
     IO.puts inspect(captures)
```

Comment:
Again with the puts'ing. I don't think we need this (was this me? doesn't matter, we can lose it).